ksql-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -252,22 +253,22 @@ public void testPrintPropertyList() {
           + "  \"@type\" : \"properties\",\n"
           + "  \"statementText\" : \"e\",\n"
           + "  \"properties\" : {\n"
-          + "    \"k1\" : 1,\n"
-          + "    \"k2\" : \"v2\",\n"
-          + "    \"k3\" : true\n"
+          + "    \"k3-KSQL\" : true,\n"
+          + "    \"k2-KSQL\" : \"v2\",\n"
+          + "    \"k1-KSQL\" : 1\n"
```

Comment:
🤔 yet again, I missed something with my suggestion - I forgot that JSON can't represent objects as map keys! The problem with this is that if anyone is depending on our HTTP CLI without having access to the java JARs deserializing this would be a little though. I apologize for the back and forth, but looking at this I think we need to switch up the model a little bit.

Instead, what do you think about modeling it as a map from `property` to a list of `(scope, value)` objects? That way, the JSON API would return:
```json
"properties": {
  "k1": [
    {"scope": "KSQL", "value": "foo1"}, 
    {"scope": "EMBEDDED CONNECT WORKER", "value": "foo2"}
  ]
}
```

let me get the opinion of some other ksqlDB committers about this (it could be that the current solution is OK)

Reply:
Alternatively, we can just return a list of `Property` instead of a map where property contains the name, value and scope (@big-andy-coates for the idea)!

Reply:
> we can just return a list of Property instead of a map where property contains the name, value and scope

@agavra I made required changes, could you please take a look? Thank you